applications/luci-app-opkg/htdocs/luci-static/resources/view/opkg.js

```diff
@@ -294,8 +294,8 @@ function display(pattern)
 		currentDisplayRows.push([
 			name,
 			ver,
-			pkg.size ? '%.1024mB'.format(pkg.size)
-			         : (altsize ? '~%.1024mB'.format(altsize) : '-'),
+			pkg.size ? '%1024mB'.format(pkg.size)
```

Comment:
Should it not be `MiB`? Also would it not be better then to convert to Mb instead?

Reply:
It took me a while to figure it out as well, but `%m` is defined as a formatting placeholder for unit prefixes, and the number determines the base. So `%1024mB` is replaced by `KiB` or `MiB`, for example, depending on the size of `pkg.size`. It would be better to convert, but as per our conversation above, I'll be making the base conversions in another commit to keep this one atomic.